src/renderers/native/ReactNativeAttributePayload.js

```diff
@@ -314,6 +318,9 @@ function diffProperties(
   for (var propKey in nextProps) {
     attributeConfig = validAttributes[propKey];
     if (!attributeConfig) {
+      if (__DEV__ && enableStrictAttributeValidation === true && propKey !== 'children') {
+        throw new Error('unsupported attribute: ' + propKey);
```

Comment:
One of our stances is that we should have the same invariants in dev & prod. Ultimately that means you never have `invariant` calls inside `__DEV__` blocks.

In ReactDOM we warn so I think that's probably the preferred thing to do here too. @vjeux, @spicyj  - could you get some RN folks to look and see what the preferred course of action is here?